test/integration/integration_test.go

```diff
@@ -45,239 +75,230 @@ func newDescriptorStatusLegacy(
+
+	configRedisSentinel(&s)
+
+	return testBasicBaseConfig(s)
 }
 
-func testBasicAuthConfigWithRedisSentinel(grpcPort, perSecond string, local_cache_size string) func(*testing.T) {
-	os.Setenv("REDIS_PERSECOND_TLS", "false")
-	os.Setenv("REDIS_PERSECOND_TYPE", "sentinel")
-	os.Setenv("REDIS_PERSECOND_URL", "mymaster,localhost:26399,localhost:26400,localhost:26401")
-	os.Setenv("REDIS_TYPE", "sentinel")
-	os.Setenv("REDIS_URL", "mymaster,localhost:26394,localhost:26395,localhost:26396")
-	os.Setenv("REDIS_TLS", "false")
+func testBasicConfigWithoutWatchRoot(perSecond bool, local_cache_size int) func(*testing.T) {
+	s := makeSimpleRedisSettings(6383, 6380, perSecond, local_cache_size)
+	s.RuntimeWatchRoot = false
 
-	return testBasicBaseConfig(grpcPort, perSecond, local_cache_size, "", "")
+	return testBasicBaseConfig(s)
 }
 
-func testBasicConfigWithoutWatchRoot(grpcPort, perSecond string, local_cache_size string) func(*testing.T) {
-	os.Setenv("REDIS_PERSECOND_URL", "localhost:6380")
-	os.Setenv("REDIS_URL", "localhost:6379")
-	os.Setenv("REDIS_AUTH", "")
-	os.Setenv("REDIS_TLS", "false")
-	os.Setenv("REDIS_PERSECOND_AUTH", "")
-	os.Setenv("REDIS_PERSECOND_TLS", "false")
-	os.Setenv("RUNTIME_WATCH_ROOT", "false")
-	os.Setenv("REDIS_TYPE", "single")
-	os.Setenv("REDIS_PERSECOND_TYPE", "single")
-	return testBasicBaseConfig(grpcPort, perSecond, local_cache_size, "", "")
+func configRedisCluster(s *settings.Settings) {
+	s.RedisPerSecondType = "cluster"
+	s.RedisPerSecondUrl = "localhost:6389,localhost:6390,localhost:6391"
+	s.RedisType = "cluster"
+	s.RedisUrl = "localhost:6386,localhost:6387,localhost:6388"
+
+	s.RedisAuth = "password123"
+	s.RedisPerSecondAuth = "password123"
+
+	s.RedisPerSecondPipelineLimit = 8
+	s.RedisPipelineLimit = 8
 }
 
-func testBasicConfigWithoutWatchRootWithRedisCluster(grpcPort, perSecond string, local_cache_size string) func(*testing.T) {
-	os.Setenv("REDIS_PERSECOND_TYPE", "cluster")
-	os.Setenv("REDIS_PERSECOND_URL", "localhost:6389,localhost:6390,localhost:6391")
-	os.Setenv("REDIS_TYPE", "cluster")
-	os.Setenv("REDIS_URL", "localhost:6386,localhost:6387,localhost:6388")
-	os.Setenv("REDIS_AUTH", "password123")
-	os.Setenv("REDIS_TLS", "false")
-	os.Setenv("REDIS_PERSECOND_AUTH", "password123")
-	os.Setenv("REDIS_PERSECOND_TLS", "false")
-	os.Setenv("RUNTIME_WATCH_ROOT", "false")
-	os.Setenv("REDIS_PERSECOND_PIPELINE_LIMIT", "8")
-	os.Setenv("REDIS_PIPELINE_LIMIT", "8")
-
-	return testBasicBaseConfig(grpcPort, perSecond, local_cache_size, "", "")
+func testBasicConfigWithoutWatchRootWithRedisCluster(perSecond bool, local_cache_size int) func(*testing.T) {
+	s := defaultSettings()
+
+	s.RedisPerSecond = perSecond
+	s.LocalCacheSizeInBytes = local_cache_size
+	s.BackendType = "redis"
+
+	s.RuntimeWatchRoot = false
+
+	configRedisCluster(&s)
+
+	return testBasicBaseConfig(s)
 }
 
-func testBasicConfigWithoutWatchRootWithRedisSentinel(grpcPort, perSecond string, local_cache_size string) func(*testing.T) {
```

Comment:
I'm noticing now that this helper function (`testBasicConfigWithoutWatchRootWithRedisSentinel`) was/is unused.